### ISSUE TITLE
Admin Menu Icon broken in 5.6

### ISSUE THREAD

**Issue by cameronjonesweb** [Automattic/jetpack#8312]
It seems the admin menu icon is broken on some sites since updating Jetpack to 5.6.
I do have WordPress installed in a subdirectory, and I spun up a test install with a conventional directory structure and it was fine so this might be the cause

![image](https://user-images.githubusercontent.com/12673074/33639826-fad61780-da78-11e7-92d6-0604f6caa533.png)

**Comment by Pseric** [comment]
Hi,

I have the same issue, and my WordPress installed in a subdirectory, too.

**Comment by jeherve** [comment]
I can't seem to be able to reproduce the issue, on a site where WordPress lives in a subdirectory. Do you use a caching service or plugin on your sites? If so, could you try to flush cache and let me know if it helps?

Thanks!

**Comment by cameronjonesweb** [comment]
Clearing my cloudflare cache seems to have fixed it.

**Comment by zigasancin** [comment]
Hi, I'm seeing the same issue, however it also happens on a root installation of WordPress. I cleaned up the server and cdn caches, but it still didn't work. Then I opened WP Admin in private browsing mode and the icon showed up as it should. Well, it seems to be a browser cache issue, because if I change the css of the icon to f102 instead of f100 in the non-private browser window, the icon shows up.

**Comment by jeherve** [comment]
> it seems to be a browser cache issue

It does indeed seem like flushing your browser cache should help.

**Comment by oskosk** [comment]
There's a strong caching mechanism for font files in browsers and in this release we swapped to a new file introducing a new sysmbol for an already used character.

Hard refreshing the page should help here:

* ctrl + shift + R in Windows/Linux.
* cmd + shift + R in Mac.

**Comment by StefMattana** [comment]
The same issue has been [reported in the JP forum](https://wordpress.org/support/topic/jetpack-dashboard-icon-disappeared/), some users have figured out already how to fix it, but I linked this issue anyway to stay on the safe side :)

**Comment by kraftbj** [comment]
Is there a way for us to force a refresh on the font files? Could we modify the build process so that, when there is a change to the jetpack.* font files, we change a query string or something, or we rename the file (jetpack-20171207.*) whenever we make changes to it?

It's not a great impression when updating to see something broken, causes support load, etc.

**Comment by htdat** [comment]
833815-zen

**Comment by smerriman** [comment]
+1 for this. Shouldn't require every WordPress user to have to clear their cache in order to not see buggy behavior.

**Comment by oskosk** [comment]
@smerriman there's no need to clear the cache. A simple [hard reload](https://www.getfilecloud.com/blog/2015/03/tech-tip-how-to-do-hard-refresh-in-browsers/) of any wp-admin page does the trick.

### Chrome:

#### Windows/Linux:

* Hold down Ctrl and click the Reload button.
* Or, Hold down Ctrl and press F5.

#### Mac:

* Hold ⇧ Shift and click the Reload button.
* Or, hold down ⌘ Cmd and ⇧ Shift key and then press R.
 
### Mozilla Firefox and Related Browsers:

#### Windows/Linux:

* Hold the Ctrl key and press the F5 key.
* Or, hold down Ctrl and ⇧ Shift and then press R.

#### Mac:

* Hold down the ⇧ Shift and click the Reload button.
* Or, hold down ⌘ Cmd and ⇧ Shift and then press R.
 

### Internet Explorer:

* Hold the Ctrl key and press the F5 key.
* Or, hold the Ctrl key and click the Refresh button.

### Safari

* Hold ⇧ Shift and click the Reload button.
* Or, hold down ⌘ Cmd and ⇧ Shift key and then press R.

**Comment by smerriman** [comment]
Yes, I know that; I was referring to the cached information that is cleared with a hard refresh, not the full browser cache. But non-developers shouldn't be required to know + perform that. WordPress and other plugins version these types of files to ensure that doesn't happen, so it should be simple for Jetpack to do the same.

**Comment by oskosk** [comment]
@smerriman Thanks for your feedback!

Jetpack does the same too. The thing here is that the font file is referenced from inside an CSS asset which is actually using the mechanism you describe to be loaded. This file is loaded as `jetpack-icons.min.css?ver=5.6` but the url of the font is not versioned inside the `jetpack-icons` file. 

As a note, this change in the logo is not something that will happen in the near future again. Probably won't happen for several years from now. 

@dereksmart what are your thoughts on this?
